site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -424,12 +422,77 @@ When enabled the user's password hash will be modified if it does not match the
 
 
 ==== Webhooks
-image::core-concepts/tenant-configuration-webhooks-settings.png[Tenant Configuration - Webhooks,width=1200,role=top-cropped]
+image::core-concepts/tenant-configuration-webhooks-settings.png[Tenant Configuration - Webhooks,width=1200,role=shadowed]
 
 ===== Table Columns
 
 include::docs/v1/tech/events-webhooks/_tenant-webhooks-table.adoc[]
 
+==== Security
+
+image::core-concepts/tenant-configuration-security-acl-catpcha.png[Tenant Configuration - Security: ACL and Captcha,width=1200,role=bottom-cropped]
+
+===== Login API settings
+
+[.api]
+[field]#Require an API key# [optional]#Optional#::
+When enabled the Login API will require an API key. This is functionally equivalent to requiring client authentication during OAuth2.
```

Comment:
This is not quite the same as the application version of this flag. This only applies when an `applicationId` is not used on the Login API. When an `applicationId` is provided, the application configuration is used.